index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Sign Up Page</title>
+  </head>
+  <body>
+    <section class="form-container">
+      <div>
+        <div class="image-wrap">
+            <img src="./images/img.jpg" alt="Green-leaf" class="green-leaf" />
+            <div class="odin">
+                <img src="./images/odin-lined.png" alt="logo-of-odin-project" class="odin-img" />
+                <p class="odin-text">ODIN</p>
+            </div>
+        </div>
+      </div>
+        <div class="forms">
+          <div class="p-group">
+              <p>
+                 This is not a real online service! You know you need something like
+                 this in your life to help you realize your deepest dreams. Sign up
+                 now to get started.
+             </p>
+             <p>You know you want to.</p>
+           </div>
+          <form action="https://formspree.io/f/xdojzrql" method="post" target="_blank">
+            <div class="form-group">
+                <h3>Let's do it! </h3>
+              <div>
+                <label for="first-name"> First Name</label> <br>
+                <input type="text" id="first-name" required /> <br />
+              </div>
+              <div>
+                <label for="last-name"> Last Name</label><br>
+                <input type="text" id="last-name" required /> <br />
+              </div>
+              <div>
+                <label for="email">Email</label><br>
+                <input type="email" name="email" id="email" required /> <br />
+              </div>
+              <div>
+                <label for="phone-number"> Phone Number</label><br>
+                <input type="tel" id="phone-number" /> <br />
+              </div>
+              <div>
+                <label for="password"> Password</label><br>
+                <input type="password" id="password" name="password" required />
+                <br />
+              </div>
+              <div>
+                <label for="confirm-password"> Confirm Password</label><br>
+                <input
+                  type="password"
+                  id="confirm-password"
+                  name="confirm-password"
+                  required
+                />
+                <br />
+              </div>
+            </div>
+            <div class="button-group">
+              <button type="submit" class="button">Create Account</button>
+              <p>
+                Already have an account?
+                <a href="#">Log in</a>
+              </p>
```

Comment:
Well done on this project.

- [ ] Please see image below: 
     - On your design, the part I label A, the space between the top line and the added image is smaller than the provided design. Kindly fix this.
     - Also, on the part label B, there should be more space between the last text and the bottom line.

Your Design | Template Design
------ | -----
![image](https://user-images.githubusercontent.com/84156714/194898646-e1895148-e95b-43a3-af7c-ee7a9dbcda99.png) |  ![image](https://user-images.githubusercontent.com/84156714/194898925-69022730-b1b3-4c23-8351-1e950c88e91c.png)